openassessment/templates/openassessmentblock/staff_debug/student_info.html

```diff
@@ -25,12 +25,11 @@ <h3 class="title">{% trans "Student Response" %}</h3>
                     {% endif %}
                 </div>
 
-                {% if submission.image_url %}
-                <img
-                    class="submission--image"
-                    alt="{% trans "The image associated with this response" %}"
-                    src="{{ submission.image_url }}"
-                />
+                {% if submission.file_url %}
+                <a href="{{ submission.file_url }}" class="submission--file">
+                    {% trans "The file associated with this response." %}
+                </a>
+                <span>(This file is uploaded by a user. Please use with caution.)</span>
```

Comment:
This text needs to be translated.  Also, "user"? Maybe "student" or "learner"?  Not sure what the edX style guide (which doesn't exist!) says about how to refer to them.